harness/config/commands.yml

```
@@ -33,7 +33,7 @@ command('networks external'):
 command('build'): |
   #!bash(workspace:/)|@
   ws external-images pull
-  passthru docker-compose build app
+  passthru docker buildx build -f ./.my127ws/docker/image/app/Dockerfile .
```

```suggestion
  passthru docker buildx bake --load
```
only while single platform image though